repo/modules.go

```diff
@@ -116,12 +119,17 @@ func importRepoRulesModules(filename string, _ *RemoteCache) (repos []Repo, err
 			log.Printf("could not determine sum for module %s", mod.Path)
 			continue
 		}
-		repos = append(repos, Repo{
+		r := Repo{
 			Name:     label.ImportPathToBazelRepoName(mod.Path),
 			GoPrefix: mod.Path,
 			Version:  mod.Version,
 			Sum:      mod.Sum,
-		})
+		}
+		if mod.Replace != nil {
+			r.Remote = mod.Replace.Path
```

Comment:
`Remote` can't be set in module mode. It only applies to VCS tools.

I think we'll need a new `replace` attribute in `go_repository` and `fetch_repo` to support this. We shouldn't need to specify the `version` and `sum` of the original module, so we can reuse those attributes for the replacement.

That should be done in a separate PR, merged before this one. Sorry, I imagine that's more complicated than you were hoping for.

Reply:
I've opened an #488 to track this.